tasks/generate.py

```diff
@@ -154,6 +154,16 @@ def parse_args(self, args):
         except ImportError:
             packages += ["setuptools"]
 
+    # Check if the user has requested us not to install wheel
+    if "--no-wheel" in args or os.environ.get("PIP_NO_WHEEL"):
```

Comment:
`--no-wheel` ?   don't you mean `--no-use-wheel`?
what about checking  `--no-binary`?